bsip-0086.md

```diff
@@ -0,0 +1,119 @@
+Valid range of that parameter is `[0, 100%]`.
+
+## When processing market fees
+
+After the protocol upgrade, when splitting a non-zero market fee, firstly
+split `amount * market_fee_network_percent` (round down) to committee-account,
+then process the remainder as before. The amount split to committee-account
+should go to the vesting balance object whose type is `market_fee_sharing`.
+
+## When checking asset authorities
+
+After the protocol upgrade, when checking authorities (E.G. white-lists) of an
+asset on an account, if the account is committee-account, let it pass.
+
+# Discussion
+
+* Other than [the "coin-days as market fees" proposal](
+ https://github.com/bitshares/bsips/issues/191) which applies
+ a discount to market fees to benefit core token holders directly, this BSIP
+ seeks for a mechanism to increase the network's income and benefit core token
+ holders indirectly.
+
+* Adjusting the fee schedule and network percent in the referral program is
+ another option to increase the network's income, and is probably sufficient
+ for funding development.
+
+* In the original design, the purpose of market fees is to reward asset
+ issuers for their work (e.g. gateway providers, or businesses built around
+ PMs), while operation fees (`limit_order_create/cancel` in particular) reward
+ the network for performing its work. The separation of market/operation fees
+ matches the separation of roles. Changing this may damage existing businesses.
+ Thus this BSIP could be a bad idea from this perspective. Also note that the
+ network can profit from market fees by setting them on committee-owned tokens.
+
+* This would be another motivation for businesses building on top of BitShares
+ to get involved in the parameter governance process. The percentage can be
+ `0` if it is more appropriate.
+
+* Asset owners can specify a zero market fee percentage to get around the
+ "tax", while still profiting by setting a higher deposit/withdrawal fee.
+ It is uncommon nowadays probably because it is less convenient or attractive
+ for their customers.
+
+* Another strategy is to charge a network market fee globally, independent of
+ what the asset owner decides. It essentially removes the option to not set
+ a fee from asset owners, which kills some freedom and would hinder non-profit
+ use of the platform.
+
+* It is possible that committee-account would accumulate a lot of tokens with no
+ value and tokens which are unable or hard to be used to pay for development.
+ It is a side effect. Hopefully there will be quite some usable tokens.
+ The committee will have to do some work to manage the tokens anyway.
+
+* The committee will be accumulating tokens which could potentially be used or
+ not used for any purpose. In other words, this BSIP as currently written does
+ not stipulate how those tokens will or will not be used by the committee.
+
+# Non-Technical Summary
+
+This BSIP adds a tool for the committee to impose a tax on market trading
```

Comment:
See [here](https://github.com/bitshares/bsips/pull/253/files#r360287036) for entire paragraph where contains word "tax".

Reply:
I need help here. Too hard for me to reword this paragraph :(

Reply:
How about this?

"This BSIP adds a tool for the committee to impose the sharing of market trading fees for every asset that has enabled a market trading fee. ..."